core/build.gradle

```diff
@@ -3,6 +3,8 @@ apply plugin: 'java-library'
 dependencies {
     compileOnly 'com.google.code.findbugs:jsr305:3.0.2'
 
+    api 'org.slf4j:slf4j-api:2.0.7'
```

Comment:
(This one does need to be an `api` dependency though, due to the public methods in `Logger` :smile:)